pygments/lexers/cplint.py

```diff
@@ -0,0 +1,54 @@
+"""
+    pygments.lexers.cplint
+    ~~~~~~~~~~~~~~~~~~~~~~
+
+    Lexer for the cplint language https://cplint.eu, including CP-logic, Logic Programs with Annotated Disjunctions, 
+    ProbLog and Distributional Clauses syntax
+
+    :copyright: Copyright 2006-2022 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import re
+
+from pygments.lexer import bygroups, inherit, words
+from pygments.lexers import PrologLexer
+from pygments.token import Text, Comment, Operator, Keyword, Name, String, \
+    Number, Punctuation
+
+__all__ = ['CplintLexer']
+
+
+class CplintLexer(PrologLexer):
+    """
+    Lexer for cplint files, https://cplint.eu, including CP-logic, Logic Programs with Annotated Disjunctions, 
+    ProbLog and Distributional Clauses syntax
+      .. versionadded:: 2.11
```

Comment:
This is missing a blank line above (Sphinx gives you a warning when compiling the docs), and has extraneous whitespace at the beginning which results in the text being indented because it gets interpreted as a quote.